src/OmniSharp.Script/ScriptHelper.cs

```diff
@@ -73,6 +74,14 @@ private CSharpCompilationOptions CreateCompilationOptions()
                 topLevelBinderFlagsProperty?.SetValue(compilationOptions, ignoreCorLibraryDuplicatedTypesValue);
             }
 
+            // in scripts, the option to supersede lower versions is ALWAYS enabled
+            // see: https://github.com/dotnet/roslyn/blob/version-2.6.0-beta3/src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs#L199
+            var referencesSupersedeLowerVersionsProperty = typeof(CompilationOptions).GetProperty(ReferencesSupersedeLowerVersionsProperty, BindingFlags.Instance | BindingFlags.NonPublic);
+            if (referencesSupersedeLowerVersionsProperty != null)
+            {
+                referencesSupersedeLowerVersionsProperty?.SetValue(compilationOptions, true);
```

Comment:
Is the `?.` needed after checking for null above?

Reply:
Technically the if could be removed couldn't it?

Reply:
you are both right 😀